packages/iov-bcp/src/transactions.ts

```diff
@@ -225,6 +225,56 @@ export interface SignedTransaction<T extends LightTransaction = UnsignedTransact
   readonly otherSignatures: readonly FullSignature[];
 }
 
+export interface ConfirmedTransaction<T extends LightTransaction> {
+  /** The transaction content. Keep this compatible to the SignedTransaction interface */
+  readonly transaction: T;
+
+  readonly height: number; // the block it was written to
+  /** depth of the transaction's block, starting at 1 as soon as transaction is in a block */
+  readonly confirmations: number;
+  /** a unique identifier (hash of the transaction) */
+  readonly transactionId: TransactionId;
+  /** application specific data from executing tx (result, code, tags...) */
+  readonly result?: Uint8Array;
+  /**
+   * Application specific logging output in an arbitrary text format that
+   * may change at any time.
+   */
+  readonly log?: string;
+  // readonly tags: ReadonlyArray<Tag>;
```

Comment:
For later inclusion?

Reply:
This code is carried around since a time when the sketched BCP types were not even compiled. Time to remove it